test/e2e/operator/operator.go

```diff
@@ -41,6 +45,7 @@ func describe() {
 	var webhook v1.Pod
 
 	ginkgo.BeforeEach(func(ctx context.Context) {
```

Comment:
I think this is looking good to me but one question: I know the namespace gets deleted after each round but the operator deployments creates non-namespaced resources too. Would it be better to have `AfterEach` then that deletes everything?

Reply:
I at first put `AfterEach` and then thought may not need.
After seeing your comment, feel like I need to bring it back.!